Mail/Template.php

```diff
@@ -234,6 +251,18 @@ public function getProcessedTemplate(array $variables = [])
         $processor->setDirectives($this->templateDirectives);
         /** @customization END */
 
+        /**
+         * After Magento v2.3.4 need set strict mode
+         *
+         * @customization START
+         */
+        if (version_compare($this->sailthruSettings->getMagentoVersion(), '2.3.4', '>=')) {
```

Comment:
By "After" in the comment do you mean including 2.3.4? If so, I think the comment should read `In Magento v2.3.4 and later, need set strict mode`